### PR TITLE
Spanish localization, and update README

### PR DESCRIPTION
**Pull request by JemAlvarez**

### CODE REVIEW

File: es.lproj/Localizable.strings [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+"Sound Alerts Disabled" = "Alertas de Sonido Deshabilitados";
+
+/* No comment provided by engineer. */
+"Sound Alerts Enabled" = "Alertas de Sonido Habilitados";
+
+/* No comment provided by engineer. */
+"Sounds" = "Sonidos";
+
+/* No comment provided by engineer. */
+"Start Tracking" = "Dejar de rastrear";
```
Comment (reubn) [inline es.lproj/Localizable.strings]: Doesn't 'dejar' mean 'stop' not 'start'?

Comment (JemAlvarez) [reply es.lproj/Localizable.strings]: Oops yes my mistake, will fix ASAP

**Comment by jordibruin** [comment]
Thanks a lot! If you have time later this week to also do the App Store text that would be even more amazing!
I forgot to commit that file 😅

**Comment by jordibruin** [comment]
@JemAlvarez I noticed I did not correctly localise 11 strings from the onboarding. I've added them to localisable file for you language if you could have a look!

**Comment by vickipetrova** [comment]
Hi again!
We added a few more sentences that are used in the new version. We’d really appreciate it if you have some time to translate those as well! 😄 @JemAlvarez

**Pull request merged**